assets/js/src/core/modules/asset/editor/types/folder/tab-manager/tabs/list/grid-container.tsx

```diff
@@ -129,6 +137,17 @@ const GridContainer = (props: GridContainerProps): React.JSX.Element => {
       columnIdentifiers.forEach((columnIdentifier) => {
         const columnIdentifierString = decodeColumnIdentifier(columnIdentifier)
 
+        item.columns?.forEach((column) => {
```

Comment:
Hmm, why exactly do we need this loop? We should have the id, isLocked and permission on row level, why do we need to set it again once for reach column?

Reply:
Afaik it needed for https://github.com/pimcore/studio-ui-bundle/pull/847/files/fec78f784453a2597e749807e17e8134438b12b5#diff-726a68b231cf47e5fc0c1e0c7a32efc200ca75bd619572faa9e5dbf423c64db2R143-R145

Besides that, we have to add the info to the `row` - we have the information in the `item`

Reply:
optimized loop: https://github.com/pimcore/studio-ui-bundle/pull/847/commits/3aff39f45047370a673d67846d2928ff62db8783

Reply:
Please remove the console.log(), then it's fine 👍